app/src/main/java/com/sdpteam/connectout/profile/Profile.java

```diff
@@ -72,32 +72,4 @@ public String getNameLowercase() {
     public enum Gender {
         FEMALE, MALE, OTHER
     }
-
-    public static class CalendarEvent {
```

Comment:
Finally this whole mess is gone 🤩

Reply:
Yes indeed, it came from a good intention, but there was obviously a better way of implementing all of this.